content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-github-classroom.md

```diff
@@ -0,0 +1,50 @@
+---
+title: About GitHub Classroom
+shortTitle: About GitHub Classroom 
+intro: '{% data variables.product.prodname_classroom %} is a space where teachers, organizations or school administrators can create assignments, collaborate with teaching assistants, and invite students in a single course.'
+product: 'Education'
```

Comment:
```suggestion
```

The `product` field is really only used for specific gated features, so this also isn't necessary 🙂